draft-ietf-rats-corim.md

```diff
@@ -2021,6 +2021,12 @@ For example, if the Evidence format is known in advance, CoRIMs using a profile
 
 Later stages will further select the CoRIMs appropriate to the Evidence Appraisal stage.
 
+#### CoRIM Trust Anchors
+The selected CoRIM's should be signed.  If so, the signatures should be validated against an endorsed public key.
```

Comment:
As we support unsigned CoRIMs in specification, Kindly refer to Section 4.3. that details handling of unsigned CoRIMs.

So I would suggest, we change the sentence to render something like below:

`When CoRIM's are signed, the signatures must be validated using correct trust anchors inside the Verifier`

Reply:
```suggestion
If CoRIM tags are signed, the signatures MUST be validated using appropriate trust anchors (certification paths) available to the Verifier.
```

I really would like to avoid the term "endorsed public key" here. Management of a Verifier's TA store is a policy decision maybe is a "very specific endorsement" that is covered by draft-ietf-rats-concise-ta-stores.